akka-remote/src/main/scala/akka/remote/artery/compress/InboundCompressions.scala

```diff
@@ -354,15 +355,17 @@ private[remote] abstract class InboundCompression[T >: Null](
     }
   }
 
-  final def confirmAdvertisement(tableVersion: Byte): Unit = {
+  final def confirmAdvertisement(tableVersion: Byte, gaveUp: Boolean): Unit = {
     tables.advertisementInProgress match {
       case Some(inProgress) if tableVersion == inProgress.version ⇒
         tables = tables.startUsingNextTable()
-        log.debug("Confirmed compression table version [{}] for originUid [{}]", tableVersion, originUid)
+        log.debug(
+          "{} compression table version [{}] for originUid [{}]",
+          if (gaveUp) "Gave up" else "Confirmed", tableVersion, originUid)
```

Comment:
This is on the destination side, which builds up the tables and announce them to the sending side. This only means that the sending side have not seen the new table and will therefore not use it. The destination side keeps several tables around so that sender can still use old table.

Reply:
Ah, thanks I missed out the `Inbound` part.